.github/workflows/ci.yml

```diff
@@ -42,6 +42,7 @@ jobs:
         with:
           name: lint-reports
           path: build/reports/ktlint
+        if: always()
```

Comment:
Is this here by mistake?

Reply:
Yes I had to clone the project again and create the branch so when i pulled everything that i had from this pr, it added @ttoino's commits that i had locally and this branch didn't have remotely.

Reply:
Is it a problem?

Reply:
That PR was already merged, so this should not be here. Can you fix that?